### PR TITLE
Updated welcome message with new committee and code of conduct

### LINKED ISSUE #269: Welcome message should point towards Code of Conduct
Since UQCS now has a code of conduct, I think it's probably a good idea if new users are pointed towards it when they join. 

A link in the welcome message could work.

### LINKED ISSUE #384: Update welcome script with new committee
;)

### PR DESCRIPTION
**Pull request by JamesDearlove**
Fixes #384
Fixes #269

### CODE REVIEW

File: uqcsbot/scripts/welcome.py [hunk 1]
```diff
@@ -10,9 +10,16 @@
     "Hey there! Welcome to the UQCS slack!",
     "This is the first time I've seen you, so you're probably new here",
     "I'm UQCSbot, your friendly (open source) robot helper",
-    "We've got a bunch of generic channels (e.g. #banter, #games, #projects) along with many " \
+    "We've got a bunch of generic channels (e.g. #banter, #memes, #projects) along with many " \
```
Comment (mitchmcdee) [inline uqcsbot/scripts/welcome.py]: would prefer to keep games and choose between either banter or memes for diversity sake 👍

**Comment by JamesDearlove** [comment]
Fixed up that channel and fixed a small typo 👍 
welcome updated

File: uqcsbot/scripts/welcome.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "Your friendly committee and admins are Nicholas Lambourne @ndl, Jack Caperon @jam, Bradley Stone @Σ, " \
     "Joshua Sutton @jsutton, Ella de Lore @Ella, Matthew Lake @mgtlake, Ishraque Zahin @Ishraque, " \
```
Comment (mitchmcdee) [inline uqcsbot/scripts/welcome.py]: wrap their slack names in brackets

File: uqcsbot/scripts/welcome.py [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "Hey there! Welcome to the UQCS slack!",
     "This is the first time I've seen you, so you're probably new here",
     "I'm UQCSbot, your friendly (open source) robot helper",
-    "We've got a bunch of generic channels (e.g. #banter, #memes, #projects) along with many " \
+    "We've got a bunch of generic channels (e.g. #banter, #games, #projects) along with many " \
     "subject-specific ones (e.g #csse1001, #math1051, #csse2310)",
     "To find and join a channel, tap on the channels header in the sidebar",
     "The UQCS slack is a community for anyone who likes code, computers and pizza so " \
```
Comment (mitchmcdee) [inline uqcsbot/scripts/welcome.py]: capitalise to `Slack`

File: uqcsbot/scripts/welcome.py [hunk 4]
```diff
@@ -19,7 +19,7 @@
     "Your friendly committee and admins are Nicholas Lambourne @ndl, Jack Caperon @jam, Bradley Stone @Σ, " \
     "Joshua Sutton @jsutton, Ella de Lore @Ella, Matthew Lake @mgtlake, Ishraque Zahin @Ishraque, " \
     "Ryan Kurz @lighthou and Raghav Mishra @rm",
-    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up"
+    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up",
```
Comment (mitchmcdee) [inline uqcsbot/scripts/welcome.py]: remove the `to see what is coming up` (redundant given the start of the sentence imo)

**Comment by JamesDearlove** [comment]
Changes have been added
welcome 2

**Pull request merged**